src/components/StakeForm.js

```diff
@@ -0,0 +1,166 @@
+import React from "react";
+
+export const StakeForm = ({ validateStake, handleChange, amount }) => (
+  <div className="card">
+    <div className="card-body">
+      <form>
+        <div className="form-group">
+          <label>Amount to Stake</label>
+          <input
+            name="amount"
+            type="number"
+            className="form-control"
+            onChange={handleChange}
+            value={amount}
+          />
+        </div>
+        <button
+          onClick={validateStake}
+          type="submit"
+          className="btn btn-primary"
+        >
+          Continue
+        </button>
+      </form>
+    </div>
+  </div>
+);
+
+export const StakeCard = ({ stakingCap, reward, stakedTotal }) => (
+  <div className="col-sm-3 top-padding">
```

Comment:
Reduce the number of html components, instead move as much as you can into structural components, to ensure core readability + consistency of UI across different pages and components. For example, <Page>..<Card>...<TextInput>...